ReSharper.FSharp/src/FSharp.Psi.Services/src/Daemon/Highlightings/FcsErrors.xml

```diff
@@ -207,6 +207,17 @@
     <QuickFix>RemoveUnnecessaryUpcastFix</QuickFix>
   </Warning>
 
+  <Error staticGroup="FSharpErrors" name="ValueNotContainedMutabilityAttributesDiffer" ID="FS0034: but its signature specifies The mutability attributes differ">
```

Comment:
It seems the error description got cut off?

Reply:
I updated it. Is my assumption correct that the error specific parts of the message are to be left out of this?

The original error was: `C:\Users\nojaf\Projects\SampleWithSigs\A.fs(3,14): error FS0034: Module 'A' contains↔    val mutable a: int    ↔but its signature specifies↔    val a: int    ↔The mutability attributes differ [C:\Users\nojaf\Projects\SampleWithSigs\SampleWithSigs.fsproj]`

The `val mutable a: int` needs to go right?